From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dyscorv <174674743+dyscorv@users.noreply.github.com>
Date: Mon, 19 Aug 2024 00:00:00 +0000
Subject: [PATCH 1/2] fix-paths


diff --git a/components.py b/components.py
index 7d29218..d20d92f 100644
--- a/components.py
+++ b/components.py
@@ -6,7 +6,7 @@ import copy
 import comfy_execution.graph_utils
 from .tools import VariantSupport
 
-comfy_path = os.path.dirname(folder_paths.__file__)
+comfy_path = os.getcwd()
 js_path = os.path.join(comfy_path, "web", "extensions")
 inversion_demo_path = os.path.dirname(__file__)
 
