From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dyscorv <174674743+dyscorv@users.noreply.github.com>
Date: Mon, 19 Aug 2024 00:00:00 +0000
Subject: [PATCH] fix-paths


diff --git a/utils.py b/utils.py
index cf3dd6e..2acdfa8 100644
--- a/utils.py
+++ b/utils.py
@@ -43,8 +43,9 @@ if os.path.exists(config_path):
             log.error("Failed to create config ckpts directory. Using default.")
             annotator_ckpts_path = str(Path(here, "./ckpts"))
 else:
-    annotator_ckpts_path = str(Path(here, "./ckpts"))
-    TEMP_DIR = tempfile.gettempdir()
+    import folder_paths
+    annotator_ckpts_path = str(Path(folder_paths.models_dir) / "controlnet_aux" / "ckpts")
+    TEMP_DIR = str(Path(folder_paths.models_dir) / "controlnet_aux" / "temp")
     USE_SYMLINKS = False
     ORT_PROVIDERS = ["CUDAExecutionProvider", "DirectMLExecutionProvider", "OpenVINOExecutionProvider", "ROCMExecutionProvider", "CPUExecutionProvider", "CoreMLExecutionProvider"]
 
